### DISCUSSION TITLE
SSL Issue when attempting $urlContent = file_get_contents

### DISCUSSION THREAD

**Discussion by TimTheEnchanta** [Codeception/Codeception#6601] [Q&A]
I receive the following error on certain sites:

**[PHPUnit\Framework\Error\Warning] file_get_contents(): SSL operation failed with code 1. OpenSSL Error messages:
error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed**

Only happens on certain sites when trying to access it with phpbrowser

**Comment by Naktibalda** [comment]
`$urlContent = file_get_contents`  is not a module-phpbrowser code.
